react/src/utils/Time.js

```diff
@@ -74,6 +77,10 @@ function FormatDuration(startDate, endDate = new Date(), verbose = false, fallba
            + (suffix ? " " + suffix : "");
 }
 
+function Ago(date, verbose = true) {
+    return FormatDuration(date, new Date(), verbose, null, null, "ago");
```

Comment:
It's been a while since I did JS, so does `new Date()` get you "now"? It might be more perspicuous to do (untested, please check carefully):
```suggestion
    var now = new Date()
    return FormatDuration(date, now, verbose, null, null, "ago");
```

Reply:
Yes, new Date() gives "now". Will make this change.